odoo/addons/base/models/ir_translation.py

```diff
@@ -770,6 +770,15 @@ def make_domain(fld, rec):
                 except AccessError:
                     pass
 
+            action.update({
+                'name': _('Translate Field'),
+                'res_model': 'translation.field.wizard',
+                'view_id': self.env.ref('base.view_translation_field_wizard').id,
+                'target': 'new',
+                'view_mode': 'form',
+                'context': {'model': model, 'id': id, 'field': field}
```

Comment:
Please avoid generic context variable. Model for what? Which id? Imagine this context variable is kept in many subcalls, it should be easy to identify even if propagated.
Can't we use the conventional `default_...` name to simplify?